doc/API.md

```diff
@@ -608,6 +613,120 @@ In axe-core v1 the main method for axe was `axe.a11yCheck()`. This method was re
 - .a11yCheck requires a context object, and so will not fall back to the document root.
 - .a11yCheck does not return a Promise.
 
+### Virtual DOM Utilities
+
+#### API Name: axe.utils.getFlattenedTree
```

Comment:
We need something to pass to `axe.utils.getNodeFromTree`, so I'd vote for documenting it and dropping `axe._tree`. But we could also do as Dylan suggests and create a method to set and clear the cache. I had some questions about that in a separate comment.